### PR TITLE
Replaced deprecated node API usage, bumped node support to version 14

### PR DESCRIPTION
**Pull request by Xetera**
Travis CI support was also updated, turns out it was because of the old ubuntu version. 
I was able to get this to compile locally. This was my first time working with the node C++ APIs, happy to try to fix things if there's anything wrong.

### CODE REVIEW

**Comment by ghost** [comment]
Last commit was in 2018, I doubt this is getting pushed.

**Comment by Xetera** [comment]
At least if other people are struggling with getting it to run they know where to find the fix

File: webp.cc [hunk 1]
```diff
@@ -5,14 +5,17 @@
 #include "cairo.h"
 #include <stdint.h>
 
+using v8::Object;
+using v8::Local;
+
```
Comment (zbjornson) [inline webp.cc:10]: Looks like these are unused.

**Review by zbjornson — COMMENTED** [review]
Thanks for working on this. I totally forgot about this module.

Can you remove package-lock.json please? That should only be checked in by end-users of the module.

File: webp.cc (hunk 1 above)
Comment (Xetera) [reply webp.cc:10]: Interesting, when I try to compile without them I get errors like
```
‘Object’ was not declared in this scope; did you mean ‘v8::Object’?
```
which is why I originally added them in. Is this something related to the node version I'm on?

Comment (zbjornson) [reply webp.cc:10]: Ah I see, I was only looking at the lines changed in this PR. Line 47L/50R uses `Local` and `Object`. 👍

**Review by zbjornson — APPROVED** [review]
Great, thank you for fixing this up!

**Pull request merged**